package.json

```diff
@@ -2306,9 +2306,9 @@
         "less-plugin-inline-urls": "^1.2.0",
         "levn": "^0.4.1",
         "lolex": "^5.1.2",
-        "lru-cache": "^7.7.3",
+        "lru-cache": "^6.0.0",
```

Comment:
7.7.3 was never used anywhere. & this was throwing errors when running `npm list --production --parseable --depth=99999 --loglevel=error` which is run by `vsce` when building the vsix package